server/test/com/sixsq/slipstream/ssclj/resources/device_dynamic_lifecycle_test.clj

```diff
@@ -77,6 +77,9 @@
                                   :wifiAddress                           "[snic(family=<AddressFamily.AF_INET: 2>, address='10.192.167.20', netmask='255.255.0.0', broadcast='10.192.255.255', ptp=None), snic(family=<AddressFamily.AF_INET6: 10>, address='fe80::fe39:ed60:dff6:db85%wlp3s0', netmask='ffff:ffff:ffff:ffff::', broadcast=None, ptp=None), snic(family=<AddressFamily.AF_PACKET: 17>, address='d4:6a:6a:9a:6b:87', netmask=None, broadcast='ff:ff:ff:ff:ff:ff', ptp=None)]"
                                   :ethernetThroughputInfo                ["0", "0", "0", "0", "0", "0", "0", "0"]
                                   :wifiThroughputInfo                    ["21689997", "950419307", "150482", "663270", "0", "0", "0", "0"]
+                                  :sensorType                            ["temperature", "humidity"]
```

Comment:
These, and the ones in the other file, should be adapted to there being multiple sensors on one device.